static/js/src/release-chart.js

```diff
@@ -41,8 +42,9 @@ import {
 function buildCharts() {
   if (document.querySelector("#server-desktop-eol")) {
     delete desktopServerStatus.MAINTENANCE_UPDATES;
-    createChart(
+    createSupportChart(
       "#server-desktop-eol",
```

Comment:
this chart is also used in two other places, namely /server and /about/release-cycle#ubuntu. If on /server it's okay to have it rebranded, on /about/release-cycle#ubuntu I think it's not okay, because there are other charts for Kernel, OpenStack and Kubernetes that are still implemented in the old way. And having one that is rebranded doesn't make sense. I think you should rebrand it only for /desktop/developers for now.